tests/integration/test_interchain.py

```diff
@@ -281,18 +349,40 @@ def test_update_bitcoin_interchain_returns_at_rest_object(self):
         jsonschema.validate(response.get("response"), schema.bitcoin_interchain_at_rest_schema)
 
     def test_update_interchains_with_nonexistant_interchain_returns_404(self):
+        # make sure that "garbage" interchains don't already exist:
+        self.client.delete_interchain_network("bitcoin", "garbage")
```

Comment:
if one of these 'garbage' interchains did exist, that would be a problem considering we only put 'garbage' in the name for interchains that should fail to create. We probably shouldn't be trying to remove these before-hand for that reason

Reply:
I had tossed in this pre-emptive cleanup because on an earlier failed run of the integration tests, I had somehow created a 'garbage' interchain, so this testcase was failed cause it did actually exist.  Good point; I will remove this addition.